src/ruby/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'rdoc', '~>4.0'
+gem 'rspec', '~> 3.0'
```

Comment:
I think there is no need to install it in runtime since generating rdoc is handled by rubygems. I use it as development_dependency in my project.